pkg/sql/catalog/systemschema/system.go

```diff
@@ -2921,9 +2921,9 @@ var (
 				Name:                "primary",
 				ID:                  1,
 				Unique:              true,
-				KeyColumnNames:      []string{"username", "path"},
```

Comment:
I think you'll probably need to write a migration that alters the schema for existing (22.2) clusters. The schema specified in this file is only used when bootstrapping a cluster for the first time.